platform/linux-generic/odp_packet.c

```diff
@@ -67,7 +67,8 @@ const _odp_packet_inline_offset_t _odp_packet_inline ODP_ALIGNED_CACHE = {
 	.input_flags    = offsetof(odp_packet_hdr_t, p.input_flags),
 	.flags          = offsetof(odp_packet_hdr_t, p.flags),
 	.subtype        = offsetof(odp_packet_hdr_t, subtype),
-	.cls_mark       = offsetof(odp_packet_hdr_t, cls_mark)
+	.cls_mark       = offsetof(odp_packet_hdr_t, cls_mark),
+	.ipsec_ctx      = offsetof(odp_packet_hdr_t, ipsec_ctx)
```

Comment:
It would be nice to have a comma at the end so next time when something is added the old lines do not need to be touched.

Reply:
Fixed in V2.